content/fr/docs/reference/kubectl/kubectl.md

```diff
@@ -41,13 +37,6 @@ kubectl [flags]
 <td></td><td style="line-height: 130%; word-wrap: break-word;">log sur l'erreur standard en plus d'un fichier</td>
 </tr>
 
-<tr>
-<td colspan="2">--application-metrics-count-limit int&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Défaut : 100</td>
-</tr>
-<tr>
-<td></td><td style="line-height: 130%; word-wrap: break-word;">Nombre max de métriques d'applications à stocker (par conteneur)</td>
```

Comment:
Why is this deleted?

Reply:
There are lots of flags that no longer exists in the `en` version of the document..